### PR TITLE
Style the Edit Profile screen header

### PR DESCRIPTION
**Pull request by amarquesmoura**
This adds styling to the Edit Profile screen header

### CODE REVIEW

**Comment by teelau** [comment]
Change the pull request title to include your initials, this helps with documentation

File: client/js/screens/EditProfile/EditProfile.js [hunk 1]
```diff
@@ -2,19 +2,42 @@ import React from "react";
 import { Text, View, TouchableOpacity, ScrollView } from "react-native";
 import EditProfileAbout from "../../components/EditProfileAbout";
 import EditProfileContact from "../../components/EditProfileContact";
+import styles from "./styles";
 
-const EditProfile = ({ toggleTab, switchTab, user }) => (
+const EditProfile = ({ toggleTab, secondTabActive, user }) => (
```
Comment (teelau) [inline client/js/screens/EditProfile/EditProfile.js:7]: This should be a stateful component, seperate the stateful logic from the Query, and move it into this component.

**Review by teelau — CHANGES_REQUESTED** [review]

File: client/js/screens/EditProfile/styles.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import theme from "../../config/globalStyles";
+
+const styles = {
+  header: {
+    marginVertical: 40,
+    marginHorizontal: 30,
```
Comment (siddharthparmar7) [inline client/js/screens/EditProfile/styles.js:6]: That's too much margin around your header. Can you use flexbox properties?

Comment (amarquesmoura) [reply client/js/screens/EditProfile/styles.js:6]: Done.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**